jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -235,4 +235,19 @@ public static void reachabilityFence(java.lang.Object ref) {
 }
 /*[ENDIF]*/
 
+/*[IF Java11]*/
+/**
+ * A Reference cannot be meaningfully cloned. Instead, construct a new Reference. This method will always
+ * throw CloneNotSupportedException. A clone of this instance will not be returned.
+ * 
+ * @return a clone of this instance
+ * 
+ * @throws CloneNotSupportedException always
+ *
+ * @since 11
+ */
+protected Object clone() throws CloneNotSupportedException {
+	throw new CloneNotSupportedException("A Reference cannot be meaningfully cloned. Instead, construct a new Reference."); //$NON-NLS-1$
```

Comment:
Please create an external message for this.

Reply:
See /jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties
and for example:
```
			/*[MSG "K0626", "Unable to handle type {0}."]*/
			throw new InternalError(com.ibm.oti.util.Msg.getString("K0626", type)); //$NON-NLS-1$
```

Reply:
added an external message, and updated description.